modules/highgui/test/test_gui.cpp

```diff
@@ -148,7 +148,9 @@ static void Foo(int, void* counter)
         && !defined HAVE_WIN32UI \
         && !defined HAVE_WAYLAND \
     ) \
-    || defined(__APPLE__)  // test fails on Mac (cocoa)
+    || defined(__APPLE__)  /* test fails on Mac (cocoa) */ \
+    || defined HAVE_FRAMEBUFFER /* trackbar is not supported */ \
+    || defined HAVE_FRAMEBUFFER_XVFB /* trackbar is not supported */
```

Comment:
`HAVE_FRAMEBUFFER_XVFB` is left here and on line 195.

Reply:
fixed